src/handlers/mod.rs

```diff
@@ -52,7 +57,7 @@ pub fn handle_app(key: Key, app: &mut App) {
                 if let Some(playing_item) = &current_playback_context.item.clone() {
                     if let Some(artist) = playing_item.artists.first() {
                         if let Some(artist_id) = &artist.id {
-                            app.get_artist_albums(artist_id, &artist.name);
+                            app.get_artist(artist_id, &artist.name);
```

Comment:
Looks like the global press `A` (`shift + a`) is not taking the user to artist.

I think you just need to add this here?

```rust
app.push_navigation_stack(RouteId::Artist, ActiveBlock::ArtistBlock);
```

Reply:
Modified relevant part following your suggestion and i works well. Now, pressing `A` can take user to the artist view.